sdk/communication/azure-communication-identity/tests/test_communication_identity_client.py

```diff
@@ -67,6 +67,72 @@ def test_create_user_and_token(self, communication_livetest_dynamic_connection_s
 
         assert user.properties.get('id') is not None
         assert token_response.token is not None
+        
+    @CommunicationPreparer()
+    def test_create_user_and_token_with_custom_minimum_validity(self, communication_livetest_dynamic_connection_string):
+        identity_client = CommunicationIdentityClient.from_connection_string(
+            communication_livetest_dynamic_connection_string,
+            http_logging_policy=get_http_logging_policy()
+        )
+        
+        token_expires_in = timedelta(minutes=60)
+        user, token_response = identity_client.create_user_and_token(scopes=[CommunicationTokenScope.CHAT], token_expires_in=token_expires_in)
+        
+        assert user.properties.get('id') is not None
+        assert token_response.token is not None
+        
+        from devtools_testutils import is_live
```

Comment:
maybe you should import it at the beginning of the file instead of importing it in each test case